doc/examples/segmentation/plot_niblack_sauvola.py

```diff
@@ -20,13 +20,23 @@
        binarization," Pattern Recognition 33(2),
        pp. 225-236, 2000.
        DOI:10.1016/S0031-3203(99)00055-2
+.. [3] C. Wolf, J-M. Jolion, "Extraction and Recognition of
+       Artificial Text in Multimedia Documents", Pattern
+       Analysis and Applications, 6(4):309-326, (2003).
+       DOI:10.1007/s10044-003-0197-7
+.. [4] Phansalskar, N; More, S & Sabale, A et al. (2011), "Adaptive
```

Comment:
This paper received 22 citations, the one above 100.  That one we can perhaps justify, but the latter?